integration/testdata/build-dependencies/app1/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+# image2 is set by Skaffold to be the image built for app2
+docker build -t $IMAGE --build-arg image2 .
+if $PUSH_IMAGE; then
```

Comment:
This is a nifty construct that I haven't seen used before.  But you're executing data that is passed in, which we generally try to avoid.

Reply:
makes sense, I pirated this code block from another existing example. I'll fix it in both places.